### DISCUSSION TITLE
Does this "new frontend" mean the appearance is different?

### DISCUSSION THREAD

**Discussion by CHNtentes** [Comfy-Org/ComfyUI#4797] [Q&A]
I found my comfyui look the same as it was... Or is it only code refinement?

![Desktop Screenshot 2024 09 05 - 23 39 11 70](https://github.com/user-attachments/assets/86ca3a5f-ddb3-41bc-948e-d4e886ecc3d6)

**Comment by JorgeR81** [comment]
The new version looks different.
The right click menu is the same, but there is a side bar and a new node search box.
There is also an optional top bar that replaces the floating panel.

You may need to update Comfy UI.

![newfe](https://github.com/user-attachments/assets/9dc4b427-e503-43d3-972e-e5fc789a08a6)

**Reply by mratashi** [reply]
How i can Run this UI?

**Reply by JorgeR81** [reply]
This new version, with the sidebar, is now the default UI.

You just need to update your Comfy UI.

You can to  that by running your `update_comfyui.bat` file.

Or you can also use the Comfy Manager extension.
https://github.com/ltdrdata/ComfyUI-Manager

**Comment by ltdrdata** [comment]
You can select `beta ui` on settings dialog.
